package.json

```diff
@@ -31,23 +31,23 @@
   "homepage": "https://github.com/dabapps/eslint-config-dabapps",
   "dependencies": {
     "eslint": "^6.8.0",
-    "eslint-config-prettier": "6.9.0",
     "eslint-plugin-prettier": "^3.1.2",
-    "prettier": "^1.19.1"
+    "prettier": "^1.19.1",
+    "typescript": "^3.7.5"
   },
   "devDependencies": {
+    "@typescript-eslint/eslint-plugin": "^2.19.0",
+    "@typescript-eslint/parser": "^2.19.0",
+    "eslint-config-prettier": "^6.10.0",
+    "eslint-plugin-jest": "^23.6.0",
+    "jest": "^25.1.0",
     "jsonlint": "^1.6.3",
     "react": "^16.12.0"
   },
-  "peerDependencies": {
-    "eslint": "^6.8.0",
-    "prettier": "1.19.1",
-    "eslint-config-prettier": "6.9.0",
-    "eslint-plugin-prettier": "3.1.2"
-  },
   "optionalDependencies": {
-    "babel-eslint": "10.0.3",
-    "eslint-plugin-react": "7.17.0",
-    "eslint-plugin-react-native": "3.8.1"
+    "babel-eslint": "^10.0.3",
+    "eslint-plugin-react": "^7.17.0",
+    "eslint-plugin-react-native": "^3.8.1",
+    "eslint-plugin-react-hooks": "^2.3.0"
```

Comment:
I think we should make all the dependencies production dependencies for ease of use (this may also be why you were having issues with the local install).

Reply:
Can you also let me know if there is any other library I am missing/not-needed here? @JakeSidSmith